man/paru.conf.5

```diff
@@ -204,8 +204,12 @@ Build packages in a chroot. This rquires the LocalRepo option to be enabled.
 Optionaly a directory may be passed to specify where the create the chroot.
 
 .TP
-.B SkipReview
+.B SkipReview [= ask] <always|never|ask>
```

Comment:
This should be:

.B SkipReview [= always|never|ask]

Reply:
Shouldn't `[= ask|always|never]`, i.e. what comes after `=` is the default when the option is not set?
This seems to be how the `RemoveMake` option is implemented/documented atleast.

```
RemoveMake [= ask]
```
```rust
"RemoveMake" => {
                let value = value.unwrap_or("yes").into();
                self.remove_make = validate(value, yes_no_ask)?;
}
```
```rust
#[default = "no"]
    pub remove_make: String,
```

Reply:
The first value is what the option is set to if set without a value.

Reply:
Hmm, okay, but then either the documentation or implementation for `RemoveMake` incorrect, right?

Reply:
No actually I just didn't think what I was saying. That's in the general case, but yes isn't an option because it's redundant.